### DISCUSSION TITLE
p/32e9550dd977485c9701f935b17f771c/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#63] [Announcements]
# p/32e9550dd977485c9701f935b17f771c/

周六时突发奇想将 Oneplus 8T 的系统升级到了最新的 Oxygen 13，结果等来的是完全换皮的国内的 ColorOS 系统，廉价的五颜六色的图标，还有各种无法关闭的微妙广告，让吾辈非常厌恶。  最终，吾辈决定刷机。但考虑到吾辈之前的小米就是被刷机变砖的，所以这次选择了付费远程支持。在简单搜索了一些内容后，主要是在 这篇文章 的推荐下，吾辈选择了 Lineage OS，它是一个开源的 An

https://blog.rxliuli.com/p/32e9550dd977485c9701f935b17f771c/

**Comment by CyrusYip** [comment]
> 但在一些简单的使用之后，意外的发现没有包含 Google 服务，这对于吾辈而言完全不可接受。

LineageOS 官方有提供谷歌服务，安装时机有讲究。要在安装 LineageOS 后，开机前安装。

https://wiki.lineageos.org/gapps/

**Reply by rxliuli** [reply]
嗯，吾辈已经习惯 pixel 系统了，下次一定（
